### PR TITLE
Use standard oclif run.js & dev.js

### PR DESCRIPTION
**Pull request by otaviojacobi**
Change-type: patch

This removes one more patch & allows us to more easily use libraries that expect the entrypoint to be `bin/run.js` (e.g auto update library). Tested with standalone, installers & npm installs.

The initial reason why we have renamed is mostly in here https://github.com/balena-io/balena-cli/pull/1285/files because before using oclif we had a `./bin/balena` which I just made become a symlink to the original run.js file.

Resolves: #   
Change-type: major|minor|patch   
Depends-on: <url>   
See: <url>   

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

File: bin/balena
Comment (otaviojacobi) [inline bin/balena:1]: Symlink so when working/debugging locally we can still do `./bin/balena`

File: bin/balena-dev
Comment (otaviojacobi) [inline bin/balena-dev:1]: Symlink so when working/debugging locally we can still do `./bin/balena-dev`

File: bin/dev.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
```
Comment (otaviojacobi) [inline bin/dev.js:1]: This is just the exact same content of previous bin/balena-dev

File: bin/run.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```
Comment (otaviojacobi) [inline bin/run.js:1]: This is just the exact same content of previous bin/balena

**Review by myarmolinsky — APPROVED** [review]

**Pull request merged**